packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -364,6 +363,10 @@ export function TransferPanel() {
       throw 'Signer is undefined'
     }
 
+    const destinationAddressError = await getDestinationAddressError({
```

Comment:
Not using the store state here. We make another call to make sure everything is correct before initiating the transfer.